index.bs

```diff
@@ -6187,18 +6182,21 @@ five forms are allowed.
 
 This section describes how definitions written with the IDL defined in
 [[#idl]] correspond to particular constructs
-in ECMAScript, as defined by the <cite>ECMAScript Language Specification 6th Edition</cite>
+in ECMAScript, as defined by the <cite>ECMAScript Language Specification</cite>
 [[!ECMA-262]].
 
 Objects defined in this section have internal properties as described in
 ECMA-262 [=sections 9.1=] and
-[=9.3.1=] unless otherwise specified, in which case one or
+[=ECMA-262 section 9.3|9.3=] unless otherwise specified, in which case one or
 more of the following are redefined in accordance with the rules for exotic objects:
-\[[Call]],
-\[[Set]],
+\[[SetPrototypeOf]],
+\[[PreventExtensions]],
+\[[GetOwnProperty]],
 \[[DefineOwnProperty]],
-\[[GetOwnProperty]], and
-\[[Delete]].
+\[[Set]],
+\[[Delete]],
+\[[OwnPropertyKeys]], and
+\[[Call]].
```

Comment:
This order came from ECMA-262: https://tc39.github.io/ecma262/#sec-object-internal-methods-and-internal-slots

Reply:
I'm wondering if we should get rid of the above list too. I'm not sure keeping it around adds any value.

Reply:
I was thinking about that too. I'll remove it.